src/core/ee/emotion.cpp

```diff
@@ -34,6 +34,22 @@ const char* EmotionEngine::REG(int id)
     return names[id];
 }
 
+const char* EmotionEngine::COP0_REG(int id)
```

Comment:
Recommend marking function as `const`.

```suggestion
const char* EmotionEngine::COP0_REG(int id) const
```

Reply:
this function is static, so it can't be `const` unfortunately